templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User: {{ user.username }}
+    </li>
+    <li>
+      <a href="{% url "logout" %}">Log out</a>
+    </li>
+  {% else %}  
+    <li><a href="{% url "login" %}">Log in</a></li>
```

Comment:
The `login` URL is correctly referenced. Ensure that the URL name `login` is correctly defined in your authentication URLs.